### PR TITLE
Update vulnerable dependencies

### PR DESCRIPTION
**Pull request by priidikvaikla**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
   "contributors": [
```
Comment (sergeikretov) [inline package.json]: remove this block completely plz, or put all contributors, cause github counted at least 10

Comment (nelsongomes) [reply package.json]: nevermind we're abandoning kardia, closed that PR

Comment (nelsongomes) [reply package.json]: we're not using kardia anymore fyi

Comment (sergeikretov) [reply package.json]: by we, you mean who?

Comment (nelsongomes) [reply package.json]: dev teams in Lisbon

Comment (sergeikretov) [reply package.json]: https://github.com/search?q=org%3Apipedrive+kardia&type=Code

Comment (sergeikretov) [reply package.json]: what do you use instead?

Comment (nelsongomes) [reply package.json]: for metrics prometheus client, for health check an endpoint in carrier

Comment (priidikvaikla) [reply package.json]: Ahhaa, will try the prometheus

**Pull request closed without merging**